openshift-examples/keycloak.yaml

```diff
@@ -102,22 +96,22 @@ parameters:
     description: The name for the application.
     value: keycloak
     required: true
-  - name: KEYCLOAK_USER
+  - name: KEYCLOAK_ADMIN
     displayName: Keycloak Administrator Username
     description: Keycloak Server administrator username
     generate: expression
     from: '[a-zA-Z0-9]{8}'
     required: true
-  - name: KEYCLOAK_PASSWORD
+  - name: KEYCLOAK_ADMIN_PASSWORD
     displayName: Keycloak Administrator Password
     description: Keycloak Server administrator password
     generate: expression
     from: '[a-zA-Z0-9]{8}'
     required: true
-  - name: DB_VENDOR
-    displayName: DB Vendor
-    description: 'DB vendor (H2, POSTGRES, MYSQL or MARIADB)'
-    value: H2
+  - name: KC_PROXY
+    displayName: Keycloak proxy configuration
+    description: 'Keycloak Proxy configuration (edge, passthrough, reencrypt, none)'
+    value: edge
```

Comment:
Does it actually make really sense to expose this? Anything but `edge` requires additional config (TLS keys mounting) which is not exposed, is that correct? If so I'd say we might need to expose it all for the proxy to be meaningful.

Alternative approach might be hardcoding a TLS Secret that is generated by OpenShift itself, similarly how it's implemented in the [RH-SSO templates](https://github.com/jboss-container-images/redhat-sso-7-openshift-image/blob/861625fa0aa1ad4c013174223566fa2b4bd34090/templates/sso75-ocp4-x509-https.json#L152).

In any case, SSL config is out of scope of this. But without SSL config, proxy config makes only little sense to me. So I'd suggest removing the proxy param and addressing it as a follow-up.